Makefile

```diff
@@ -50,8 +50,8 @@
 
 #Add iso code for any locales you want to support here (space separated)
 # default is no locales
-# LOCALES = af
-LOCALES = cs de el en es eu fi fr gl hu it nl no pl pt pt_BR ro ru sl sv
+LOCALES = fr de el en es eu fi gl it pl pt ru sv
```

Comment:
Why removing pt_BR ?

Reply:
I would say it's a mistake. They are many places where a list of languages is specified but they are not all the same. In this makefile, in transifex, in the script to compile strings.

@rldhont According to the Python code, Lizmap takes only the first 2 letters, never the country. So `pt_BR` is the same as `pt`: https://github.com/3liz/lizmap-plugin/blob/master/lizmap.py#L124 So AFAIK `pt_BR` has never been used. I just tried indeed:
![Screenshot from 2019-09-20 07-21-15](https://user-images.githubusercontent.com/1609292/65301401-9641da00-db77-11e9-9c60-b6e38cbe41cc.png)

Watch what Lizmap is loading in the log panel and what was my language. This would need a fix.